db/repository/category.py

```diff
@@ -0,0 +1,39 @@
+from db.base import User, Category
+from sqlalchemy.orm import Session
+from fastapi import HTTPException,status
+
+def create(request, db: Session,current_user):
```

Comment:
i think current_user is already User obj, u don't need to query again. please check this pls

Reply:
Sorry my bad.... I type hinted as User in route but it is actually TokenData (services/JWTtoken.py). When I am writing that since Read and Read By ID doesn't need user object so I don't want to query it on every endpoints so I query it in necessary endpoints but I am looking for a way to change that common code into decorator in the future.

Reply:
ok